src/Nox.Generator/Application/Integration/EtlCreatedEvent.template.cs

```diff
@@ -0,0 +1,18 @@
+//Generated
+
+#nullable enable
+
+using MediatR;
+using Nox.Integration.Abstractions.Interfaces;
+using Nox.Integration.Abstractions.Models;
+
+namespace {{codeGeneratorState.ApplicationNameSpace}}.Integrations;
+
+public class {{ className }}: NoxEtlRecordCreatedEvent<INoxEtlEventPayload>, INotification
```

Comment:
to keep consistent with the rest of generated code, I suggest the following changes to this and apply the same convention to other files:
```suggestion
public class {{ className }}: EtlRecordCreatedEvent<IEtlEventDto>, INotification
```

Reply:
Implemented